qemu/tests/cfg/uefi_check_boot_option.cfg

```diff
@@ -0,0 +1,37 @@
+- uefi_check_boot_option:
+    only q35
+    only ovmf
+    type = uefi_check_boot_option
+    start_vm = no
+    boot_menu = on
+    del boot_once
+    del boot_order
+    del boot_strict
+    variants:
+        - splash_time_10:
+            boot_splash_time = 10000
+            expected_result = "10"
```

Comment:
If the output likes: SetVariable(Timeout, 10)?  Just set "10", is it unique in logs?

Reply:
Yes,thanks for your patience,I will modify it.